superset/db_engine_specs/presto.py

```diff
@@ -423,24 +423,30 @@ def get_function_names(cls, database: Database) -> List[str]:
         return database.get_df("SHOW FUNCTIONS")["Function"].tolist()
 
     @classmethod
-    def _partition_query(  # pylint: disable=too-many-arguments,too-many-locals
+    def _partition_query(  # pylint: disable=too-many-arguments,too-many-locals,unused-argument
         cls,
+        schema: Optional[str],
         table_name: str,
+        indexes: List[Dict[str, Any]],
         database: Database,
         limit: int = 0,
         order_by: Optional[List[Tuple[str, bool]]] = None,
         filters: Optional[Dict[Any, Any]] = None,
     ) -> str:
-        """Returns a partition query
+        """
+        Return a partition query.
 
+        Note the unused arguments are exposed for sub-classing purposes where custom
```

Comment:
@michael-s-molina per our offline discussion I added this comment as to why the unused arguments are included.